docs/ja/README.md

```diff
@@ -0,0 +1,130 @@
+<p align="center">
+ <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://github.com/atuinsh/atuin/assets/53315310/13216a1d-1ac0-4c99-b0eb-d88290fe0efd">
+  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="https://github.com/atuinsh/atuin/assets/53315310/08bc86d4-a781-4aaa-8d7e-478ae6bcd129">
+</picture>
+</p>
+
+<p align="center">
+<em>魔法のシェル履歴</em>
+</p>
+
+<hr/>
+
+<p align="center">
+  <a href="https://github.com/atuinsh/atuin/actions?query=workflow%3ARust"><img src="https://img.shields.io/github/actions/workflow/status/atuinsh/atuin/rust.yml?style=flat-square" /></a>
+  <a href="https://crates.io/crates/atuin"><img src="https://img.shields.io/crates/v/atuin.svg?style=flat-square" /></a>
+  <a href="https://crates.io/crates/atuin"><img src="https://img.shields.io/crates/d/atuin.svg?style=flat-square" /></a>
+  <a href="https://github.com/atuinsh/atuin/blob/main/LICENSE"><img src="https://img.shields.io/crates/l/atuin.svg?style=flat-square" /></a>
+  <a href="https://discord.gg/Fq8bJSKPHh"><img src="https://img.shields.io/discord/954121165239115808" /></a>
+  <a rel="me" href="https://hachyderm.io/@atuin"><img src="https://img.shields.io/mastodon/follow/109944632283122560?domain=https%3A%2F%2Fhachyderm.io&style=social"/></a>
+  <a href="https://twitter.com/atuinsh"><img src="https://img.shields.io/twitter/follow/atuinsh?style=social" /></a>
+  <a href="https://actuated.dev/"><img alt="Arm CI sponsored by Actuated" src="https://docs.actuated.dev/images/actuated-badge.png" width="120px"></img></a>
+</p>
+
+
+[English] | [简体中文] | [日本語]
+
+
+Atuinは、既存のシェル履歴をSQLiteデータベースに置き換え、コマンドの追加コンテキストを記録します。さらに、Atuinサーバーを介して、マシン間で履歴を完全に暗号化して同期するオプションも提供します。
+
+
+
+
+<p align="center">
+  <img src="demo.gif" alt="animated" width="80%" />
+</p>
+
+<p align="center">
+<em>終了コード、実行時間、時刻、コマンドが表示されます</em>
+</p>
+
+
+
+
+
+検索UIに加えて、次のようなこともできます：
```

Comment:
This sentence doesn't seem to make sense, but the original English text "*As well as the search UI, it can do things like this*" is strange too. @ellie Could you check the English text?